packages/fields/src/components/es-checkbox/es-checkbox.tsx

```diff
@@ -28,7 +28,11 @@ export class EsCheckbox {
 
     render() {
         return (
-            <label class={{ field: true, invalid: !!this.invalid }}>
+            <label
+                class={{ field: true, invalid: !!this.invalid }}
+                tabindex={1}
```

Comment:
Should probably be `0`, as we want it to be "focusable in sequential keyboard navigation".

![image](https://github.com/user-attachments/assets/b6647a87-46b5-4205-96f0-66e84397f366)

Doesn't actually matter in this case, each shadowDOM has it's own sequence, so `1` here means the first thing focusable within the showRoot, but `0` shows the intent better.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex